com.unity.render-pipelines.core/Runtime/Common/CommandBufferPool.cs

```diff
@@ -17,12 +17,14 @@ public static class CommandBufferPool
         public static CommandBuffer Get()
         {
             var cmd = s_BufferPool.Get();
-            cmd.name = "Unnamed Command Buffer";
```

Comment:
Should we leave this unchanged? I suppose this is to make default nameless command buffers appear on profiler?
@JulienIgnace-Unity any concern with changing?

Reply:
Yeah I think the goal was to have a marker by default. But since we now know that it can cause issues implicitly in some case it's probably better to have it unnamed by default.

Reply:
that ^, I guess for now this should suffice but probably in the future we can work together to find out what kind of construct would best suit both srps in terms of gathering profiling data :)